doc/FAQ

```diff
@@ -628,9 +628,9 @@ Starting game.
    blocking the connection.
 
    If you are running a personal firewall, make sure that you allow
-   communication for freeciv-server and the Freeciv client to the trusted
+   communication for freeciv21-server and the Freeciv client to the trusted
```

Comment:
... and the Freeciv21 client...

Reply:
The FAQ is full of obsolete info and should be cleaned up in a separate PR. This is a result of an automated search and replace.